runtime/battery-station/src/lib.rs

```diff
@@ -101,32 +112,92 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     state_version: 1,
 };
 
+#[derive(scale_info::TypeInfo)]
+pub struct ContractsCallfilter;
+
+// Currently disables Court, Rikiddo and creation of markets using Court or SimpleDisputes
+// dispute mechanism.
+impl Contains<RuntimeCall> for ContractsCallfilter {
+    fn contains(runtime_call: &RuntimeCall) -> bool {
+        #[allow(clippy::match_like_matches_macro)]
+        match runtime_call {
+            RuntimeCall::AssetManager(transfer { .. }) => true,
+            // Membership is managed by the respective Membership instance
```

Comment:
This comment seems incorrect.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1041/commits/7e82892d656ea1f0e56205b84c02662a4ef09a24